frontend/src/features/mapSideBar/tabs/SectionField.tsx

```diff
@@ -14,7 +16,16 @@ export const SectionField: React.FunctionComponent<ISectionFieldProps> = props =
   return (
     <Row className={props.className ?? 'pb-2'}>
       <Col xs={props.labelWidth ?? '4'} className="pr-0 text-left">
-        <StyledFieldLabel>{props.label}:</StyledFieldLabel>
+        <StyledFieldLabel>
```

Comment:
slightly concerned about the possibility of the tooltip dropping onto the next line here, although I suppose that may be better then the alternative.

Reply:
IMO is better to keep it responsive so if it has a long label text it can drop to the next line. If we force it to be in the same line then we could potentially have forms that look odd as columns wouldn't be aligned anymore. Will leave as-is and we can always change if users report bugs against it